### PR TITLE
frontend: cleanup translations for add and manage accounts

### PR DESCRIPTION
**Pull request by thisconnect**
- removed old unused translations
- TODO: remove from locize as well
- moved translations for adding to addAccount, same for manageAccounts
- added missing translations (also on settings view)

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -1039,28 +1047,15 @@
...
     },
-    "success": {
-      "message": "<strong>{{accountName}}</strong> has now been added to your accounts.",
-      "nextButton": "Done",
-      "step": "Finished",
-      "title": "Account added"
-    },
-    "title": "Manage accounts"
+    "title": "Manage accounts",
+    "settingsButton": "$t(manageAccounts.title)",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json:1057]: delete this and just use the other key in the source code? locize charges also by number of keys, and translators will have to deal with this too.

**Review by benma — APPROVED** [review]

File: frontends/web/src/locales/en/app.json [hunk 2]
```diff
@@ -39,19 +39,27 @@
   },
   "addAccount": {
     "accountName": "Account name",
-    "address": "Address",
-    "coin": "Coin",
+    "choose-name": {
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:42]: is that a question? :)

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:1057]: > locize charges also by number of keys

🙈

File: frontends/web/src/locales/en/app.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:42]: can we keep it? 

see https://github.com/digitalbitbox/bitbox-wallet-app/blob/staging-multiaccounts/frontends/web/src/routes/account/add/add.tsx#L214

[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/locales/en/app.json:42]: okay, but that has another drawback: to be able to filter all keys that are in use in the future, it is great to have the t(...) strings be literals and not contain variables :shrug:

[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:42]: I find it elegant as it is now, but maintainability wins so I'll rewrite it.

**Review by benma — APPROVED** [review]
utACK, nice!

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -39,19 +39,27 @@
...
       "alreadyExists": "The account already exists.",
       "limitReached": "Cannot add account. The maximum number of accounts for this coin has been reached."
     },
-    "extendedPublicKey": "Extended public key",
-    "scriptType": "Script type",
-    "submit": "Add account",
-    "title": "Add account",
-    "warning": {
-      "xpubWrongNet": "Warning: The extended public key format indicates that it might not be valid for the selected account type. If you do not see the expected transactions, then try to import the extended public key with another account type."
-    }
```
Comment (thisconnect) [inline frontends/web/src/locales/en/app.json:54]: @benma this is the old translations

**Pull request merged**